CONTRIBUTING.md

```diff
@@ -45,23 +50,31 @@ Here are the current repos for the gonum project.  If code you want to contribut
 * [talks](https://github.com/gonum/talks) - Talks given about Go and Gonum delivered by Gonum developers.
 * [gonum.github.io](https://github.com/gonum/gonum.github.io) - Gonum website
 
-## How to Contribute
+## Contributing
 
 ### Reporting Bugs
 
-When you encounter a bug, please open an issue on the corresponding repo.  Be as specific as possible.  If you are able to write a test that reproduces the bug, please open a pull instead of an issue.  As a rule we keep all tests OK.
+When you encounter a bug, please open an issue on the corresponding repo.  Start the issue title with the repo/subrepo name, like `stat/distmv: Issue Name`.  Be specific about the environment you encountered the bug in.  If you are able to write a test that reproduces the bug, please include it in the issue.  As a rule we keep all tests OK.
 
 ### Suggesting Enhancements
 
 If the scope of the enhancement is small, open an issue.  If it is large, such as suggesting a new repo, sub-repo, or interface refactoring, then please start a discussion on the gonum-dev list.
 
 ### Your First Code Contribution
 
-Before your first merge, you will need to be added to the [CONTRIBUTORS](https://github.com/gonum/license/blob/master/CONTRIBUTORS) and [AUTHORS](https://github.com/gonum/license/blob/master/AUTHORS) file.  Open a pull request adding yourself to them.
+If you are a new contributor, thank you!  Before your first merge, you will need to be added to the [CONTRIBUTORS](https://github.com/gonum/license/blob/master/CONTRIBUTORS) and [AUTHORS](https://github.com/gonum/license/blob/master/AUTHORS) file.  Open a pull request adding yourself to them.
+
```

Comment:
We should have a comment on licensing. Something like "all gonum code follows the BSD license in the license document. It is preferred that code contributions do not come with additional licensing. For exceptions, added code must also follow a BSD license"